packages/web/src/components/nav/desktop/NowPlayingArtworkTile.tsx

```diff
@@ -121,7 +122,7 @@ export const NowPlayingArtworkTile = () => {
       kind='track'
       id={trackId}
       isOwner={isOwner}
-      link={permalink}
```

Comment:
Not sure what we thought we were using this field as before; just permalink doesn't go anywhere for browser

Reply:
interesting, there was some casing logic before but i think that also was not doing the full link
https://github.com/AudiusProject/audius-client/pull/3137/files#diff-1e991467df13ad8a00f0c358f72094f2551641944d3b4eab44942744ba992e35L546

weird